### DISCUSSION TITLE
Having issue on LDAP configuration

### DISCUSSION THREAD

**Discussion by TahaAydas** [provectus/kafka-ui#3435] [Q&A] [answered]
Hello guys, I'm kinda like rookie on Provectus. I have set up the provectus image on my company OCP cluster succesfully but I also wanna add to LDAP conf. When I initialize the service, I face the issue as like that;

[30m2023-03-03 16:57:56,411[0;39m [34mINFO [0;39m [[34mmain[0;39m] [33mo.s.l.c.s.AbstractContextSource[0;39m: Property 'userDn' not set - anonymous context will be used for read-write operations

Plus, I'm also adding my envrionment variables;

![image](https://user-images.githubusercontent.com/33178817/222784048-2fd3b829-03c0-496c-8a8b-a4358b7cb2cb.png)

Thank for your support in advance

**Comment by Haarolean** [comment]
hi, that's a WARN level, not an issue, basically.
Which problem are you facing?

**Reply by TahaAydas** [reply]
Actually when I entered my ldap information on pop-up which is appear on browser, it doesn't accept that my credentials is true. So, I only found only this clue about that.

**Reply by Haarolean** [reply, marked as answer]
@TahaAydas unfortunately I've heard a few times about such issues but I can't even debug them without a reproducible scenario. 
We use this compose [example](https://github.com/provectus/kafka-ui/blob/master/documentation/compose/auth-ldap.yaml) for development and we can successfully login with these users. If you provide a reproducible scenario, we'll happily take a look.
